rotkehlchen/chain/aggregator.py

```diff
@@ -1450,37 +1455,54 @@ def check_chains_and_add_accounts(
             address=account,
             chains=chains,
         )
-        new_tracked_chains = self.track_evm_address(account, active_chains)
+        new_tracked_chains, failed_chains = self.track_evm_address(account, active_chains)
 
-        return [(chain, account) for chain in new_tracked_chains]
+        return (
+            [(chain, account) for chain in new_tracked_chains],
+            [(chain, account) for chain in failed_chains],
+        )
 
     def add_accounts_to_all_evm(
             self,
             accounts: list[ChecksumEvmAddress],
-    ) -> list[tuple[SUPPORTED_EVM_CHAINS, ChecksumEvmAddress]]:
+    ) -> tuple[
+        list[tuple[SUPPORTED_EVM_CHAINS, ChecksumEvmAddress]],
+        list[tuple[SUPPORTED_EVM_CHAINS, ChecksumEvmAddress]],
+        list[tuple[SUPPORTED_EVM_CHAINS, ChecksumEvmAddress]],
+    ]:
         """Adds each account for all evm chain if it is not a contract in ethereum mainnet.
 
-        Returns a list of tuples of the address and the chain it was added in.
+        Returns three lists:
+        - list of tuples of the address and the chain it was added in.
+        - list of tuples of the address and the chain that already existed.
+        - list of tuples of the address and the chain that failed to be queried.
 
         May raise:
         - RemoteError if an external service such as etherscan is queried and there
         is a problem with its query.
         """
+        all_evm_chains = get_args(SUPPORTED_EVM_CHAINS)
         added_accounts: list[tuple[SUPPORTED_EVM_CHAINS, ChecksumEvmAddress]] = []
+        failed_accounts: list[tuple[SUPPORTED_EVM_CHAINS, ChecksumEvmAddress]] = []
+        already_existing_addresses: list[tuple[SUPPORTED_EVM_CHAINS, ChecksumEvmAddress]] = []
```

Comment:
Please be consistent. Use either accounts or addresses. And here since it's combo of address + chain, use accounts

```suggestion
        already_existing_accounts: list[tuple[SUPPORTED_EVM_CHAINS, ChecksumEvmAddress]] = []
```